documentation/staging/content/userguide/managing-operators/conversion-webhook.md

```diff
@@ -0,0 +1,112 @@
+        service:
+          name: weblogic-operator-webhook-svc
+          namespace: sample-weblogic-operator-ns
+          path: /webhook
+          port: 8084
+      conversionReviewVersions:
+      - v1
+```
+
+#### Install the conversion webhook
+Beginning with  Operator version 4.0, when you [install the operator]({{<relref "/userguide/managing-operators/installation#install-the-operator">}}) using the `helm install` command with the operator Helm chart, it also configures a deployment and supporting resources for the conversion webhook and deploys the conversion webhook in the specified namespace. However, the CRD conversion strategy can point to only a single conversion webhook, and the Kubernetes cluster can have only one active conversion webhook. Therefore if the conversion webhook deployment already exists in some other namespace, then a new conversion webhook deployment is not created.
+
+{{% notice note %}}
+The cluster-level permissions for listing and reading the Namespaces and Deployments are needed to search all the namespaces for existing conversion webhook deployment. In addition, permissions to create and read the events in the conversion webhook namespace are needed for the conversion webhook service account.
+{{% /notice %}}
+
+
+If you want to **only** install the conversion webhook (and not the Operator) in the given namespace, set the Helm configuration value `webhookOnly` to `true` in the `helm install` command. After meeting the [prerequisite requirements]({{<relref "/userguide/managing-operators/preparation.md">}}), call:
+```
+$ helm install sample-weblogic-conversion-webhook \
+  weblogic-operator/weblogic-operator \
+  --namespace sample-weblogic-conversion-webhook-ns \
+  --set "webhookOnly=true" \
+  --wait
+```
+The above command creates a Helm release named `sample-weblogic-conversion-webhook` in the `sample-weblogic-conversion-webhook-ns` namespace, configuring a deployment and supporting resources for the conversion webhook and deploying the conversion webhook.
+
+To check if the conversion webhook is deployed and running,
+see [Troubleshooting]({{<relref "/userguide/managing-operators/troubleshooting#check-the-conversion-webhook-deployment">}}).
+
+To prevent the removal of conversion webhook during the `helm uninstall` command, set the Helm configuration value `preserveWebhook` to `true` during installation using the `helm install` command. 
+
+The following table describes the behavior of different `Helm` commands (with the Operator Helm chart) and the Helm configuration values:
+
+| Helm command with the operator Helm chart | Helm configuration values | Behavior |
+| --- | --- | --- |
+| Helm install | Default | Operator and conversion webhook installed. |
+| Helm install | Default and existing `weblogic-operator-webhook` deployment | Operator installed. |
+| Helm install | `webhookOnly=true` | Conversion webhook installed. |
+| Helm install | `preserveWebhook=true` | Operator and webhook installed. |
```

Comment:
Operator and webhook installed --> Operator and webhook installed and a future uninstall will not remove the webhook.